API_doc/Libconfig.h

```diff
@@ -0,0 +1,53 @@
+//
+// Must be included to use the LibConfig
+//
+//
+
+#pragma once
+
+# include <stddef.h>
+
+#ifdef _cplusplus
+namespace LibConfig
+{
+#endif
+
+  typedef void (*onChangeNotifier)(void *data, char const *newValue);
```

Comment:
Types usually have a capitale name, and function typedefs should have a prefix or suffix helping to differentiate them.